emb_tau_id_sfs_ul.sh

```diff
@@ -0,0 +1,666 @@
+            --setParameters ES_${dm_cat}=0.2,r=0.9 --setParameterRanges r=${min_id},${max_id}:ES_${dm_cat}=${min_es},${max_es} \
+            --robustFit=1 --setRobustFitAlgo=Minuit2  --X-rtd FITTER_NEW_CROSSING_ALGO --X-rtd FITTER_NEVER_GIVE_UP \
+            --cminFallbackAlgo Minuit2,Migrad,0:0.001 --cminFallbackAlgo Minuit2,Migrad,0:0.01 --cminPreScan \
+            --redefineSignalPOIs ES_${dm_cat},r \
+            --floatOtherPOIs=1 --points=400 --algo grid
+
+        echo "[INFO] Moving scan file to datacard folder ..."
+        mv higgsCombine.nominal_${dm_cat}.MultiDimFit.mH120.root output/$datacard_output_dm/htt_mt_${dm_cat}/
+
+        echo "[INFO] Plotting 2D scan ..."
+        echo "[INFO] Input file: " output/$datacard_output_dm/htt_mt_${dm_cat}/higgsCombine.nominal_${dm_cat}.MultiDimFit.mH120.root
+        python3 plot_2D_scan.py --name nominal_${dm_cat} --in-path output/$datacard_output_dm/htt_mt_${dm_cat}/ \
+         --tau-id-poi ${dm_cat} --tau-es-poi ES_${dm_cat} --outname ${dm_cat} 
+    done
+
+fi
+
+
+fix_es=-1.4
+fix_id=1.097
+
+probl_nuisance="ES_DM1"
+
+probl_dm_categories=("DM1")
+
+if [[ $MODE == "PROBL_NUIS_SCAN" ]]; then
+
+    source utils/setup_cmssw_tauid.sh
+
+   echo "[INFO] Create 1D scan for problematic nuisance parameter ${probl_nuisance}"
+
+           for dm_cat in "${probl_dm_categories[@]}"
+    do
+
+    combineTool.py -M MultiDimFit -n .nominal_${dm_cat}_check_poi -d output/$datacard_output_dm/htt_mt_${dm_cat}/ws_scan_${dm_cat}.root \
+    --setParameters ES_${dm_cat}=${fix_es},r=${fix_id} --setParameterRanges r=0.5,1.5:ES_${dm_cat}=-4.0,4.0 \
+    --robustFit=1 --setRobustFitAlgo=Minuit2  --X-rtd FITTER_NEW_CROSSING_ALGO --X-rtd FITTER_NEVER_GIVE_UP \
+    --cminFallbackAlgo Minuit2,Migrad,0:0.001 --cminFallbackAlgo Minuit2,Migrad,0:0.01 --cminPreScan \
+    --redefineSignalPOIs ${probl_nuisance} \
+    --floatOtherPOIs=1 --points=400 --algo grid -v 2
+
+    echo "[INFO] Moving scan with problematic nuisance parameter file to datacard folder ..."
+
+    mv higgsCombine.nominal_${dm_cat}_check_poi.MultiDimFit.mH120.root output/$datacard_output_dm/htt_mt_${dm_cat}/
+
+    echo "[INFO] Plotting 1D scan ..."
+
+    python3 plot1DScan.py  output/$datacard_output_dm/htt_mt_${dm_cat}/higgsCombine.nominal_${dm_cat}_check_poi.MultiDimFit.mH120.root \
+     --POI $probl_nuisance --y-max 12 --output ${TAG}_${ERA}_${CHANNEL}_${WP}_${dm_cat}_${probl_nuisance}_scan
+
+    mv ${TAG}_${ERA}_${CHANNEL}_${WP}_${dm_cat}_${probl_nuisance}_scan.* output/$datacard_output_dm/htt_mt_${dm_cat}/
+    done
+fi
+
+min_id_dm0=0.8
+max_id_dm0=1.2
+min_es_dm0=-2
+max_es_dm0=2
+id_dm0=0.95
+es_dm0=-1
+
+
+min_id_dm1=0.9
+max_id_dm1=1.2
+min_es_dm1=-3.8
+max_es_dm1=0.5
+id_dm1=1.12
+es_dm1=-1.4
+
+
+min_id_dm10_11=0.9
+max_id_dm10_11=1.2
+min_es_dm10_11=-3
+max_es_dm10_11=1.5
+id_dm10_11=1.02
+es_dm10_11=-1
```

Comment:
see above (also this maybe needs some documentation what these parameters are used for)